lang/cs/Org.Apache.REEF.Client/Avro/YARN/AvroYarnJobSubmissionParameters.cs

```diff
@@ -60,6 +60,12 @@ public static string Schema
         public string jobSubmissionDirectoryPrefix { get; set; }
 
         /// <summary>
+        /// Gets or sets file system url
```

Comment:
Yes, it will be set to yarnConfiguration for the property fs.defaultFS. I will have a separate PL for it after this one is merged. 
I will add comments for it.